video_xblock/video_xblock.py

```diff
@@ -51,37 +58,79 @@ class VideoXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.content,
     )
 
+    player_id = String(
+        default='default',
+        display_name=_('Player Id'),
+        help=_('Your Brightcove player id. Use "Luna" theme for all your players'),
```

Comment:
Remove "Your".
Is user indented to use "Luna" theme, or theme is used in this XBlock?